packages/notebook/src/browser/contributions/notebook-actions-contribution.ts

```diff
@@ -59,6 +59,12 @@ export namespace NotebookCommands {
     });
 }
 
+// for commands where the html title differs from button label
+export const NOTEBOOK_COMMAND_TITLES: {[key: string]: string} = {
+    'notebook.add-new-markdown-cell': nls.localizeByDefault('Add Markdown Cell'),
+    'notebook.add-new-code-cell': nls.localizeByDefault('Add Code Cell')
+};
```

Comment:
Suggestion: This doesn't seem like an extensible way of setting tooltips for specific commands.

[vscode simply uses](https://github.com/microsoft/vscode/blob/8ba046adbeecbbabac874d1a9b6b1a41f8cedf19/src/vs/workbench/contrib/notebook/browser/controller/insertCellActions.ts#L320-L325) a `tooltip` property on their commands which is displayed in the toolbar tooltip. We should probably do something similar.

Reply:
I was actually thinking about enhancing the command interface but then I thought it is maybe too much just for one case. But ok. Will do so.

Reply:
It doesn't necessarily need to adjust the command interface. We have added special interfaces for that [in the past](https://github.com/eclipse-theia/theia/blob/9a51317fe81a179e9e7c6c9071e9a00218bba299/packages/notebook/src/common/notebook-common.ts#L25).